operator-common/src/main/java/io/strimzi/operator/common/model/Labels.java

```diff
@@ -48,22 +48,26 @@ public class Labels {
     public static final String STRIMZI_KIND_LABEL = STRIMZI_DOMAIN + "kind";
 
     /**
-     * The Strimzi cluster the resource is part of.
-     * The value is the cluster name (i.e. the name of the cluster CM)
+     * The Strimzi cluster the resource is part of. This is typically the name of the custom resource.
      */
     public static final String STRIMZI_CLUSTER_LABEL = STRIMZI_DOMAIN + "cluster";
 
     /**
-     * The name of the K8S resource.
-     * This is often the same name as the cluster
-     * (i.e. the same as {@code strimzi.io/cluster})
-     * but is different in some cases (e.g. headful and headless services)
+     * Name of the Strimzi component to which given resource belongs. E.g. Kafka or ZooKeeper. This label does not
+     * depend on the name of the cluster. This is useful to identify resources which belong to the same component but
+     * different clusters which is useful for example for scheduling (e.g. when you do not want this broker to be
+     * scheduled on a node where any other Kafka broker is running).
+     */
+    public static final String STRIMZI_COMPONENT_LABEL = STRIMZI_DOMAIN + "component";
```

Comment:
is this the label for the component type? I am a little bit lost across ... component, component type and application :-(